resources/hab_build.rb

```diff
@@ -84,6 +91,12 @@ def last_build_env
     live_stream new_resource.live_stream
     sensitive true
   end
+
+  file 'artifact' do
```

Comment:
`file 'remove-artifact'` or something clear as to what the artifact resource here is.